### DISCUSSION TITLE
CockroachDB returns QueryResultType<ExternalQueryResultRowType>

### DISCUSSION THREAD

**Discussion by delaneyj** [mmkal/slonik-tools#299] [Q&A]
Per the example was able to get working in Postgres no problem.  Goes from `QueryResultType<any>` to proper type.  However even though the types appear to be generated correctly the result when using Cockroach always seem to be `QueryResultType<ExternalQueryResultRowType>`.  Any ideas how to fix this?

**Comment by mmkal** [comment]
@delaneyj this is a postgres only library, being based on slonik. I've never used cockroach so can't speak to what would/wouldn't work.
